build.gradle

```diff
@@ -0,0 +1,109 @@
+plugins {
```

Comment:
sbt is scala specific, while gradle is more versatile. I chose gradle because I'm more comfortable with it, even for scala projects, but I can migrate to SBT without a problem